modules/swagger-core/src/main/java/io/swagger/jackson/ModelResolver.java

```diff
@@ -245,13 +245,13 @@ public Model resolve(JavaType type, ModelConverterContext context, Iterator<Mode
                 if (member != null) {
                     String altName = member.getName();
                     if (altName != null) {
-                        if (altName.startsWith("get")) {
-                            if (!Character.isUpperCase(altName.charAt(3))) {
-                                propName = altName;
-                            }
-                        } else if (altName.startsWith("is")) {
-                            if (!Character.isUpperCase(altName.charAt(2))) {
+                        final int length = altName.length();
+                        for (String prefix : Arrays.asList("get", "is")) {
+                            final int offset = prefix.length();
+                            if (altName.startsWith(prefix) && length > offset
+                                    && !Character.isUpperCase(altName.charAt(offset))) {
                                 propName = altName;
+                                break;
```

Comment:
`break` statements in Java make me sad :) I have a couple of questions about the PR, but they'll wait for next week.